src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <>
+    <h2 className="Person__name">
+      My name is
+      {` ${person.name}`}
+    </h2>
+    {person.age && (
+    <p className="Person__age">
+      I am
+      {` ${person.age}`}
+    </p>
+    )}
+    {person.isMarried === true
+      ? (
+        <p className="Person__partner">
+          {`${person.partnerName} `}
+          is my
+          {person.sex === 'f' ? ' husband' : ' wife'}
+        </p>
+      ) : (<p className="Person__partner">I am not married</p>)}
```

Comment:
don't forget about code style
```suggestion
   const partner = person.sex === 'f' ? ' husband' : ' wife';
 ...
    <h2 className="Person__name">
      {`My name is ${person.name}`}
    </h2>
    
    {person.age && (
      <p className="Person__age">
        I am
        {` ${person.age}`}
      </p>
    )}
    
    {person.isMarried
      ? (
        <p className="Person__partner">
          {`${person.partnerName} is my ${partner}`}
        </p>
      ) : (
        <p className="Person__partner">I am not married</p>
      )}
```